src/server/utilities/logger.js

```diff
@@ -1,25 +1,23 @@
-const chalk = require('chalk')
-const winston = require('winston')
+const { createLogger, format, transports } = require('winston')
+const { combine, timestamp, printf } = format
 
-const tsFormat = () => new Date().toLocaleTimeString()
+const logFormat = printf(({ level, message, timestamp }) => {
+  return `${timestamp} ${level}: ${message}`
+})
 
-const log = new winston.Logger({
+const log = createLogger({
   level: process.env.LOG_LEVEL || 'info',
-  transports: [
-    new winston.transports.Console({
-      timestamp: tsFormat,
-      colorize: true,
-      prettyPrint: true
-    })
-  ]
+  format: combine(timestamp(), logFormat),
+  transports:
+    typeof process.env.TAPESTRY_LOGS_FOLDER !== 'undefined'
```

Comment:
Sorry... can we update this to `LOG_DIRECTORY` or `LOG_DIR` to be more consistent with `LOG_LEVEL`?